packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -68,7 +68,9 @@ class _SnapshotItemContent extends StatelessWidget {
                     children: [
                       Expanded(
                         child: Markdown(
-                          data: _snapshotDocumentation,
+                          data: _snapshotDocumentation(
+                            Theme.of(context).isDarkTheme,
```

Comment:
This will likely be stale when the theme changes (unless there is some rebuild higher up in the tree). Have you tried changing the theme and verifying that this updates as expected?

Also, we should use `preferences.dartModeTheme` here instead for consistency with the rest of the codebase

Reply:
I am sure we redraw everyting when theme changes. And, yes, i tested it and it works.